internal/service/service.go

```diff
@@ -0,0 +1,17 @@
+package service
+
+import (
+	"github.com/arturzhamaliyev/Flight-Bookings-API/internal/repository"
+)
+
+// Service provides user-related functionality
+type Service struct {
+	User *Users
+}
```

Comment:
why do you created generic struct for the service?
service locator is anti pattern, it's breaking Single Responsibility Principe